### PR TITLE
HPCC-19767 Configuring Log Visualization

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

Only material to review in the ConfiguringHPCC.xml file are **lines 1747 - 1791** 
Please ignore the other highlighted lines which are a result of white space and/or word wrap having used various editors on the XML. 

@kunalaswani please review
@rpastrana please review
@JamesDeFabia please review

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

View Successful Test build at http://10.240.32.243/job/DocDev70-1/

### CODE REVIEW

File: initfiles/componentfiles/configxml/espsmcservice.xsd.in [hunk 1]
```diff
@@ -202,10 +202,10 @@
                 </xs:appinfo>
             </xs:annotation>
         </xs:attribute>
-        <xs:attribute name="logStashAdresses" type="xs:string" use="optional" default="">
+        <xs:attribute name="logStashAddresses" type="xs:string" use="optional" default="">
             <xs:annotation>
```
Comment (JamesDeFabia) [inline initfiles/componentfiles/configxml/espsmcservice.xsd.in]: @rpastrana @kunalaswani  This attribute is referenced in esp/services/ws_elk/WsELKService.cpp in the misspelled form

File: initfiles/componentfiles/configxml/espsmcservice.xsd.in [hunk 2]
```diff
@@ -224,10 +224,10 @@
                 </xs:appinfo>
             </xs:annotation>
         </xs:attribute>
-        <xs:attribute name="elasticSearchAdresses" type="xs:string" use="optional" default="">
+        <xs:attribute name="elasticSearchAddresses" type="xs:string" use="optional" default="">
             <xs:annotation>
```
Comment (JamesDeFabia) [inline initfiles/componentfiles/configxml/espsmcservice.xsd.in]: @rpastrana @kunalaswani This attribute is referenced in esp/scm/ws_elk.ecm and  esp/services/ws_elk/WsELKService.cpp in the misspelled form

**Comment by g-pan** [comment]
@rpastrana  @kunalaswani  please review latest changes to source files. Please ensure the platform still builds and functions as expected. 
@JamesDeFabia good catch.

**Comment by rpastrana** [comment]
@g-pan changes look reasonable, and verified
@richardkchapman spelling issues caught in the doc review required XSD,XSL,ECM and minor CPP changes. The code changes were tested and verified.

**Pull request merged**